File: 7kyu 1/disemvowel.js
/*
|--------------------------------------------------------------------------
| Disemvowel Trolls
|--------------------------------------------------------------------------
|
*/

function disemvowel(str) {
   let string = "";
   for (let i = 0; i < str.length; i++) {
      if (str[i] != "a" && str[i] != "i" && str[i] != "u" && str[i] != "e" && str[i] != "o" &&
         str[i] != "A" && str[i] != "I" && str[i] != "U" && str[i] != "E" && str[i] != "O") {
         string += str[i]
      }
   }
   return string;
}


function disemvowel(str) {
   return str.replace(/[aeiou]/gi, '');
}


function disemvowel(str) {
   const vowels = 'aeiou';
   return str.split('').filter(letter => !vowels.includes(letter.toLowerCase())).join('');
}


console.log(disemvowel("Febrina Alivia"),
   "Fbrn lv")
console.log(disemvowel("This website is for losers LOL!"),
   "Ths wbst s fr lsrs LL!")

File: Python/7py 1/disemvowel.py
"""
|--------------------------------------------------------------------------
| Disemvowel Trolls
|--------------------------------------------------------------------------
|
"""

def disemvowel(str):
   string = ""
   for i in range(len(str)):
      if (str[i] != "a" and str[i] != "i" and str[i] != "u" and str[i] != "e" and str[i] != "o" and str[i] != "A" and str[i] != "I" and str[i] != "U" and str[i] != "E" and str[i] != "O"):
         string += str[i]
   return string


def disemvowel(s):
   return s.translate(None, "aeiouAEIOU")


def disemvowel(string):
   return "".join(c for c in string if c.lower() not in "aeiou")


import re
def disemvowel(string):
   return re.sub(r"[aeiouAEIOU]", "", string)


print(disemvowel("This website is for losers LOL!"), "Ths wbst s fr lsrs LL!")